### PR TITLE
Basic Network analysis with Road_blocks data

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: source/docs/3/basic_network_analysis.rst [hunk 1]
```diff
@@ -57,16 +57,15 @@ Procedure
...
   
-6. In the :guilabel:`Expression string builder` dialog, expand the :guilabel:`Fields and Values` section in the middle-panel. Select ``DIRECTIONA`` attribute and click :guilabel:`All Unique` in the right-hand panel. The 4 values that we discussed earlier will appear. Having these values here as a reference helps when building the expression. Also, you can double-click on any value to add them to the expression.
+6. In the :guilabel:`Expression string builder` dialog, expand the :guilabel:`Fields and Values` section in the middle-panel. Select the ``SUMMARYDIR`` attribute and click :guilabel:`All Unique` in the right-hand panel. The 4 values that we discussed earlier will appear. Having these values here as a reference helps when building the expression. Also, you can double-click on any value to add them to the expression.
 
   .. image:: /static/3/basic_network_analysis/images/6.png
     :align: center
   
 7. The goal is to create an expression that selects all one-way streets. Enter the following expression and click :guilabel:`OK`.
 
   .. code-block:: none
```
Comment (spatialthoughts) [inline source/docs/3/basic_network_analysis.rst:67]: code block is not visible. Add a blank line after .. code-block:: none

File: source/docs/3/basic_network_analysis.rst [hunk 2]
```diff
@@ -16,28 +16,28 @@ Get the data
...
+2. You will see a new line layer called ``Roadway_Block`` added to the :guilabel:`Layers` panel. This layer represents each road in Washington DC. Select the :guilabel:`Identify` tool in the :guilabel:`Attributes Toolbar`. Click on any road segment to see what attributes are attached to it. There are standard attributes like Route-name, Road-type etc. there is an attribute called ``SUMMARYDIR``. This is an import attribute for routing as it specifies whether the segment is two-way or one-way. It contains 4 different values. ``BD`` (Both Directions) for two-way streets. ``OB``  (Digitizing direction) for one-way streets where the traffic is allowed in the direction of the line (start-point to end-point) and ``IB`` (Against digitizing direction) for one-way streets where the traffic flows in the opposite direction of the line. There is also ``?? `` value where we will assume two-way traffic. We will now use the information in that attribute to display an arrow on one-way streets.
```
Comment (spatialthoughts) [inline source/docs/3/basic_network_analysis.rst:40]: Change the OB/IB description as follows , instead of digitizing direction

OB (Outbound)
IB (Inbound)

Comment (spatialthoughts) [inline source/docs/3/basic_network_analysis.rst:40]: ``?? `` is not appearing correctly in the output. Try removing space after ??. You may have to escape the characters for it to appear properly.

**Review by spatialthoughts — COMMENTED** [review]
Added a few comments. Please rupdate.

File: source/docs/3/basic_network_analysis.rst (hunk 1 above)
Comment (Santhosh-M31) [reply source/docs/3/basic_network_analysis.rst:67]: Updated. Now visible

File: source/docs/3/basic_network_analysis.rst (hunk 2 above)
Comment (Santhosh-M31) [reply source/docs/3/basic_network_analysis.rst]: Updated as said

Comment (Santhosh-M31) [reply source/docs/3/basic_network_analysis.rst]: Updated. Now it is appearing correctly.

**Pull request merged**